tlatools/org.lamport.tlatools/test/tla2sany/corpus/string.txt

```diff
@@ -146,3 +146,29 @@ op == "foo ` bar"
   (operator_definition (identifier) (def_eq) (string))
 (double_line)))
 
+==================|||
+Multiline String Literal (ish)
+==================|||
+
+---- MODULE Test ----
+op == "before `
+and anything
+" \
+in-between
+' after"
```

Comment:
Let's count the escape chars together:

1. Newline right after the backtick
2. Newline after "and anything"
3. Double quote
4. Backslash
5. Newline after backslash
6. Newline after "in-between"

For the total of 6, which is exactly the amount below 👇

Reply:
Are you able to run this test from Eclipse, or whatever Java IDE you're using? These are originally tree-sitter tests so you can also try to run this against the grammar: https://github.com/tlaplus-community/tree-sitter-tlaplus/

Reply:
I was using the ant incantation from the readme to run the paraer corpus test.

Was going to follow up with the tree-sitter change once this is in. I also have doubts they support this accidental discovery. But keeping the corpus in sync is a worthy idea for sure.